### ISSUE TITLE
[Supporting the Platform - Overview]: 'Color Indicator' button color is not visible in High Contrast Black/White theme. 

### ISSUE THREAD

**Issue by Shubham786786** [highcharts/highcharts#15439]
User Experience:
If any image , icon , hyperlink , text or visual focus disappear when high contrast mode is enabled , will make the UI confusing and difficult for users with low vision to navigate easily to the desired element.

Test Environment:
OS build Version 2004 (OS Build 20277.1)
Browser Version: Microsoft Edge Dev Version 89.0.723.0
URL: [https://www.highcharts.com/demo/bar-basic](url)

Pre-Requisites:
1. Go to settings of your system.
2. Ease of Access High contrast
3. Turn on the high contrast
4. Switch between high contrast black and white mode

Repro Steps:
1. Open above URL on Anaheim Dev Browser and login with valid credentials.
2. Observe the issue.

Actual Result:
Color Indicator buttons color is not visible in High Contrast Black/White theme.

Expected Result:
Color Indicator buttons should be visible in their original color in High Contrast Black/White theme.

MAS Reference:
[https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B3870F7C4-7C43-43C6-A9F7-FD73D800BFD0%7D&file=MAS%204.3.1%20%E2%80%93%20No%20Disruption%20of%20Accessibility%20Features.docx&action=default&mobileredirect=true&cid=9d9bb798-5332-4a2f-8eb0-b28337cd89d9](url)

Attachments:
[High Contrast Black Mode.zip](https://github.com/highcharts/highcharts/files/6262144/High.Contrast.Black.Mode.zip)
[High Contrast White Mode.zip](https://github.com/highcharts/highcharts/files/6262145/High.Contrast.White.Mode.zip)
[Normal Mode of screenshot.zip](https://github.com/highcharts/highcharts/files/6262146/Normal.Mode.of.screenshot.zip)

**Comment by KacperMadej** [comment]
Hi @Shubham786786 

Is this a problem with the website? Where is the 'Color Indicator' button?

I cannot log to the MAS Reference link you have provided (probably some account or login error). Could you upload files or provide info here - as you have already provided 3 zip files I wonder is there anything relevant on the linked MAS Reference site?

**Comment by Shubham786786** [comment]
> Hi @Shubham786786
> 
> Is this a problem with the website? Where is the 'Color Indicator' button?
> 
> I cannot log to the MAS Reference link you have provided (probably some account or login error). Could you upload files or provide info here - as you have already provided 3 zip files I wonder is there anything relevant on the linked MAS Reference site?

Hi @KacperMadej ,

MAS link is not working due to account login issue only, I am attaching a word file of the content present inside the link here
[MAS 4.3.1 – No Disruption of Accessibility Features.docx](https://github.com/highcharts/highcharts/files/6301242/MAS.4.3.1.No.Disruption.of.Accessibility.Features.docx)

And for the 'Color indicator' button, you can refer to the zipped files added in the issue.

Let me know, if any other help is required

Thanks

**Comment by KacperMadej** [comment]
@Shubham786786 
Sorry, but I still don't understand what and where is the 'Color Indicator' button - could you share an image with a big red circle or something that could help me see the problematic element?

**Comment by Shubham786786** [comment]
> @Shubham786786
> Sorry, but I still don't understand what and where is the 'Color Indicator' button - could you share an image with a big red circle or something that could help me see the problematic element?

Hey, I am attaching files here for the 'Color indicator' graphs, having red boundaries in normal mode and also attached screenshot when the settings have been applied. 
We want that the color should not fade away when high contrast settings have ben applied.
[Normal Mode of screenshot.zip](https://github.com/highcharts/highcharts/files/6308139/Normal.Mode.of.screenshot.zip)
[High Contrast Black Mode.zip](https://github.com/highcharts/highcharts/files/6308157/High.Contrast.Black.Mode.zip)

**Comment by KacperMadej** [comment]
Thank you for the details and explanation.

This is by design. When high contrast mode is detected the [high contrast theme](https://api.highcharts.com/highcharts/accessibility.highContrastTheme) is applied.
Color for each series is forced through the series update (color is based on plotOptions.<seriesType>.color with a fallback to `'windowText'`), so this allows setting the same color for all points and series (of the same type) to one color - demo: https://jsfiddle.net/BlackLabel/6pd3k8qy/3/
However, for a high contrast mode named special high contrast colors should be used (like `'windowText'`) to match the OS settings as the actual colors could be changed in the OS settings.

Keeping original colors would violate the high contrast idea and is currently not supported when using the accessibility module.
As a workaround, you could not use the accessibility module and all colors will not be changed in high contrast mode.

@Shubham786786 Please let us know if the problem is now resolved and the issue could be closed or provide more info for the dicussion.

**Comment by oysteinmoseng** [comment]
Closing this for now, note that the High contrast theme can be configured at https://api.highcharts.com/highcharts/accessibility.highContrastTheme
